frontend/package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "proxy": "http://localhost:20001",
```

Comment:
IIRC, we don't commit changes to the proxy setting here

Reply:
My bad...